src/mon/MonClient.cc

```diff
@@ -709,6 +709,9 @@ void MonClient::_finish_auth(int auth_err)
   if (!auth_err && active_con) {
     ceph_assert(auth);
     _check_auth_tickets();
+  } else if (auth_err == -EAGAIN && !active_con) {
+    ldout(cct,10) << __func__ << " auth completed with error, reopen the session" << dendl;
```

Comment:
Can we rephrase this? Maybe something like

```auth returned an error, reopening the session to try again```

Reply:
Done